### PR TITLE
Az.ApiManagement update to sdk 8.0.0 / Api Version 2021-8-01

### PR DESCRIPTION
**Pull request by wpooley**
## Description

Updating APIM to dot net sdk 8.0.0 and updating all corresponding cmdlet parameters 

## Checklist

- [X] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [X] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [X] For public API changes to cmdlets:
    - [X] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [X] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/ApiManagement/ApiManagement/ChangeLog.md [hunk 1]
```diff
@@ -19,6 +19,51 @@
 -->
 ## Upcoming Release
 
+[Breaking change] Sample has been replaced by Examples in
```
Comment (isra-fel) [inline src/ApiManagement/ApiManagement/ChangeLog.md:22]: Due to the length limit of the overall changelog of powershell modules, please make it consise.
I think we should keep the one about breaking changes, and summarize the rest with 1 sentence.

**Review by solankisamir — APPROVED** [review]

**Comment by solankisamir** [comment]
@isra-fel can you suggest what is pending in this PR?

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**